### DISCUSSION TITLE
@updateURL Question

### DISCUSSION THREAD

**Discussion by fmhnm** [quoid/userscripts#675] [General]
Hi everyone,

I’m new to userscripts, so I apologize if this is a basic question. I’m trying to figure out how to get my script to update automatically. I’ve hosted it on GIST and would like it to refresh whenever I make changes there.

I assumed that the application would do this automatically and periodically if I have the @updateURL set, but maybe I haven’t waited long enough. Do I need to manually click the update button on each of my devices, or is there a way to automate this process?

Thanks!
FM

**Comment by ACTCD** [comment]
Refer to the [#333](https://github.com/quoid/userscripts/issues/333) in the issue tracker, we haven't implemented it yet.
